capstone/fitness/src/test/java/com/google/sps/SchedulerTest.java

```diff
@@ -0,0 +1,148 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+import com.google.sps.util.*;
+import java.util.*;
+import org.junit.*;
+import static org.junit.Assert.*;
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.model.EventDateTime;
+import com.google.api.services.calendar.model.Event;
+import static org.hamcrest.Matchers.*;
+
+public class SchedulerTest { 
+  long duration = 30;
+  Scheduler scheduler= new Scheduler(duration);
+
+  
+  public static long getMillisecondsBetween(EventDateTime a, EventDateTime b){
```

Comment:
Seems like something that would go into the Time class instead. Also, the test file should focus on having tests (at least I believe so, not too sure).

Reply:
Lu wanted me to move it here since it's only used for tests.

Reply:
okay